h/accounts/__init__.py

```diff
@@ -96,3 +100,8 @@ def includeme(config):
     config.include('.schemas')
     config.include('.subscribers')
     config.include('.views')
+
+    secret = config.registry.settings['secret_key']
+    derived = derive_key(secret, b'h.accounts')
```

Comment:
I don't think the `b` is necessary here? Byte strings are the default in Python 2